packages/shared/src/components/MainLayout.tsx

```diff
@@ -91,17 +93,14 @@ export default function MainLayout({
             : 'non-responsive-header'
         }`}
       >
-        <Link
+        <LinkWithTooltip
```

Comment:
Just throwing this one out here:
Is there no way to have the SimpleTooltip determine if the direct child is a `<Link>` and then do something else?

Think it would be super cool if we could just have 1 sole component for the tooltip usage.

If it's not possible, this seems like a good first step!

Reply:
That's a cool idea. Though the part where it got me separating the two is the way we will be handling the trigger element itself. The other one, we will be wrapping the children, and the other one will be utilizing the `ref` of the actual element since `next/link` has problems with accepting custom components.

Combining them in a single component might provide quite a bit of complexity (as having multiple conditions whether to apply a ref or not). To keep things simple and easier to understand for each scenario, I had separated them.